src/Compilers/CSharp/Portable/Symbols/Source/SourceEventAccessorSymbol.cs

```diff
@@ -54,10 +55,14 @@ internal abstract class SourceEventAccessorSymbol : SourceMemberMethodSymbol
 
             this.MakeFlags(
                 isAdder ? MethodKind.EventAdd : MethodKind.EventRemove,
+                RefKind.None,
                 @event.Modifiers,
                 returnsVoid: false, // until we learn otherwise (in LazyMethodChecks).
+                hasAnyBody: false,
```

Comment:
>hasAnyBody: false, [](http://example.com/codeflow?start=16&length=18)

I think we should pass the value that reflects the state of affairs. Perhaps, derived type should pass it in. For example ```SynthesizedEventAccessorSymbol``` it looks like should have a body unless it is abstract.

Reply:
I've moved to the approach where the derived type passes this in.